server/application/logs.go

```diff
@@ -39,6 +39,11 @@ func parseLogsStream(podName string, stream io.ReadCloser, ch chan logEntry) {
 		timeStampStr := parts[0]
 		logTime, err := time.Parse(time.RFC3339Nano, timeStampStr)
 		if err != nil {
+			if strings.HasPrefix(line, "unable to retrieve container logs for ") ||
+				strings.HasPrefix(line, "Unable to retrieve container logs for ") {
```

Comment:
Replace with `strings.HasPrefix(strings.ToLower(line), "unable to retrieve container logs for ")`